### ISSUE TITLE
The SRCC of official pretrained_model(mobilenet_0.625_0.583_0.553_0.525_0.785_0.762_0.748_0.723_0.783_0.806.pth) is 0.783, which is much better than the 0.735 mentioned in paper. Would you please describe the difference between them?

### ISSUE THREAD

**Issue by kjohew** [HuiZeng/Grid-Anchor-based-Image-Cropping#20]

**Comment by HuiZeng** [comment]
Hi, we received the same question from one reviewer during our journal submission. I just copy our response as follows:

"The major reason is that we changed the deep learning toolbox from MatConvNet (in the conference version) to PyTorch (in the journal version) and all codes are rewritten using Python and Cuda. There are many different implementation details between these two toolboxes and the pretrained VGG16 models are also different. The RGB values are scaled into the range [0,1] for the VGG16 in PyTorch while the RGB values are not scaled in VGG16 in MatConvNet. The groundtruth scores are also normalized in our PyTorch implementation."

**Comment by kjohew** [comment]
Get it. Thanks a lot for the prompt reply!
